populator-machinery/controller.go

```diff
@@ -687,14 +689,15 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 		}
 
 		if err := c.checkIntreeStorageClass(pvc, storageClass); err != nil {
-			klog.V(2).Infof("Ignoring PVC %s/%s: %s", pvcNamespace, pvcName, err)
+			klog.V(4).Infof("Ignoring PVC %s/%s: %s", pvcNamespace, pvcName, err)
 			return nil
 		}
 
 		if storageClass.VolumeBindingMode != nil && storagev1.VolumeBindingWaitForFirstConsumer == *storageClass.VolumeBindingMode {
 			waitForFirstConsumer = true
 			nodeName = pvc.Annotations[annSelectedNode]
 			if nodeName == "" {
+				klog.V(4).InfoS("Waiting for node selection due to WaitForFirstConsumer mode", "pvc", klog.KObj(pvc))
```

Comment:
Why use klog.InfoS instead of klog.Infof? I think structured log can take up more storage space than the plain